File: Odoo/GenteFitOdoo/GenteFitOdoo/Resources/OddoRepositories/DTO/producto.py
''' Clase para implementar el objeto de transición entro Odoo y JSON'''


class ProductoData:

    # Constructor
    def __init__(self, id, name, default_code, list_price, standard_price, categ_id):
        self.id = id
        self.name = name
        self.default_code = default_code
        self.list_price = list_price
        self.standard_price = standard_price

        self.categ = None

        if categ_id:
            self.categ = categ_id[1] if categ_id[1] else None

File: Odoo/GenteFitOdoo/GenteFitOdoo/Models/Producto.cs
namespace GenteFitOdoo.Models
{
    public class Producto
    {
        public int? Id { get; set; }
        public string? DefaultCode { get; set; }
        public string? Name { get; set; }
        public string? Categ { get; set; }
        public decimal? ListPrice { get; set; }
        public decimal? StandardPrice { get; set; }

        // Constructor vacío
        public Producto()
        {
        }


    }
}
